src/templates/Footer/index.js

```diff
@@ -31,24 +31,25 @@ function Footer(props) {
   const {
     className,
     phoneNumber,
-    text,
+    emailAddress,
```

Comment:
Are we sure we want to require exposing an email address with `mailto:` over having a generic onclick? 

- explicit exposed email addresses on a site thats search engine crawl-able will always render that mailbox useless with the sheer amount of spam.
- allows sending email via a third-party like intercom

If we allow one or the other or both the `onClickChat` prop should either be removed or made to work.